pkg/kubernetes/manifest/deployment.go

```diff
@@ -93,6 +93,33 @@ func (d *Deployment) UpdatePodTemplateAnnotations(key, value string) error {
 	return nil
 }
 
+func (d *Deployment) UpdateImageTag(deploymentImageName, latestTag string) error {
+	if len(d.Spec.Template.Spec.Containers) == 1 {
```

Comment:
deal with if len(conatiners) == 0. I.e. invalid deployment file{reason: no images in pod spec}